### ISSUE TITLE
Hunter: The Reckoning [564E0004]

### ISSUE THREAD

**Issue by hotzenplotz3000** [Cxbx-Reloaded/game-compatibility#353]
[Hunter: The Reckoning](https://en.wikipedia.org/wiki/Hunter:_The_Reckoning_(video_game))

### Description

Game is booting and plays intros with noisy and choppy sound.
For a brief moment you can see the main menu, but then it crashes to ms dashboard.
I couldn't make a log when this happens, because it overwrites the current kernel log

### Dumps 

[Xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1303919/Xbe.txt)
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1303921/KrnlDebug.txt)
[hlecache.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1303923/hlecache.txt)

### Status
**Previous behavior:** -

**Current behavior:** booting

### Screenshots 
![capture20170914193317952](https://user-images.githubusercontent.com/30478365/30445078-3a020dcc-9985-11e7-9f05-971ac4c1d658.png)
![capture20170914193119349](https://user-images.githubusercontent.com/30478365/30445082-3bf6ddc4-9985-11e7-8bde-49bb724087a8.png)

**Comment by ObiKKa** [comment]
Almost 1 day ago @PatrickvL showed this image of this game's logo image working and appearing above the emulator window.
[
![hunter_logo](https://user-images.githubusercontent.com/18428108/30451304-8c0da6f2-99d6-11e7-89ce-d67c58468530.png)
](url)

**Comment by hotzenplotz3000** [comment]
Yeah exactly. I found this issue while testing this Game and @PatrickvL fixed it with PR https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/pull/729 👍

**Comment by ObiKKa** [comment]
This game needs fresh testing on the current Master branch to confirm if it does get in-game.

Shows a few intro splash screens but then the FMV cinematic loaded and looks a bit glitched and sized smaller towards the top-left corner of the screen. Menu is good.
The first in-game rendered cutscene ran very slowly. Shows very good graphics & performance in-game with an opaque rectangle occasionally covering the top-left area of screen.
- [Cxbx-Reloaded Xbox Emulator - Hunter: The Reckoning Ingame! 720p + FXAA! (090663d7 + WIP)](https://www.youtube.com/watch?v=mpVQ3oj5nts&list=PL4PD8cV5ypNDPTJbrol7A-5rvuteHWWYj&index=1&t=142s) (John GodGames Emus. Published on Mar 15, 2018. Has 5 WIPs.)
